core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -634,6 +635,14 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     ConditionalWhenAboveAmountResource("when above [amount] [resource]", UniqueTarget.Conditional),
     ConditionalWhenBelowAmountResource("when below [amount] [resource]", UniqueTarget.Conditional),
 
+    // These two conditionals support only stockpileable resources (Gold, Faith, Culture, Science)
+    ConditionalWhenAboveAmountStat("has more than [amount] [stat]", UniqueTarget.Conditional),
```

Comment:
This doesn't parse in an English sentence - please rename to 'when above', 'when below'

Reply:
Use the existing conditional for resource, checking if the param is a known resource. If yes, according to resource, if not, according to stat, if the unique has a stat

Reply:
The game speed modified version should take stat only, not resource

Reply:
I've made some changes requested by you. Can you review them?